packages/webdriver/src/driver/utils.ts

```diff
@@ -114,6 +115,49 @@ export async function setupChrome(caps: Capabilities.Capabilities, cacheDir: str
     return { executablePath, buildId, platform }
 }
 
+export async function setupChromedriver(chromedriverOptions: InstallOptions, cacheDir: string, platform: BrowserPlatform, buildId: string){
+    let chromedriverBinaryPath = computeExecutablePath({
+        browser: Browser.CHROMEDRIVER,
+        buildId,
+        cacheDir
+    })
+    async function hasChromedriverInstalled(chromedriverBinaryPath: string) {
+        return await fsp.access(chromedriverBinaryPath).then(() => true, () => false)
+    }
+    // Check if specified Chromedriver version is installed
+    const installOptions: InstallOptions & { unpack?: true } = {
+        ...chromedriverOptions,
+        cacheDir,
+        platform,
+        buildId,
+        browser: Browser.CHROMEDRIVER,
+        unpack: true,
+        downloadProgressCallback: (downloadedBytes: number, totalBytes: number) => downloadProgressCallback('Chromedriver', downloadedBytes, totalBytes)
+    }
+    if (!(await hasChromedriverInstalled(chromedriverBinaryPath))) {
+        // Check if specified version of Chromedriver is available for download
+        const isCombinationAvailable = await canDownload(installOptions)
+        // If specified chromedriver version is not installed and there is no specified version to download, work with the latest stable version
+        if (!isCombinationAvailable){
+            const stableChromedriverBuildId = await resolveBuildId(Browser.CHROMEDRIVER, platform, ChromeReleaseChannel.STABLE)
```

Comment:
This assumes that the user wants to test on a stable release. What happens this error appears for an older version?